org.kde.akregator.json

```diff
@@ -0,0 +1,474 @@
+{
+    "id": "org.kde.akregator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "akregator",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--own-name=org.freedesktop.Akonadi",
+        "--own-name=org.freedesktop.Akonadi.*",
+        "--own-name=org.freedesktop.Akonadi.Control.*",
+        "--own-name=org.freedesktop.Akonadi.Agent.*",
+        "--own-name=org.freedesktop.Akonadi.Resource.*",
```

Comment:
isn't the first one enough

Reply:
Last three indeed seem to be redundant. So, I'll have these two :
```
"--own-name=org.freedesktop.Akonadi",
"--own-name=org.freedesktop.Akonadi.*",
```

I'm supposing those two don't imply each other.